.gitlab-ci.yml

```
@@ -303,3 +308,13 @@ linting:
   rules:
     - if : '$CI_PIPELINE_TRIGGERED == null'
 
+github-release:
+  stage: deploy
+  image: alpine:latest
+  tags: [docker]
+  rules:
+    - if: '$CI_COMMIT_TAG != null'
```

```suggestion
    - if: '$CI_COMMIT_TAG =~ /^v\d+\.\d+\.\d+$/'
```

Escaping I have no idea :p